client/src/pages/OrganizationProfile.js

```diff
@@ -1,6 +1,6 @@
 import { WhiteSpace } from "antd-mobile";
 import axios from "axios";
-import React, { useState, useEffect, useContext } from "react";
+import React, { useState, useEffect, useContext, Text } from "react";
```

Comment:
are you using this Text Anywhere @MohamedEl1 ? If not, remove it lol Again, this PR will have like a 10000 commits xD we could've done it local but it's ok, we can squash and merge so we don't clutter master with all this commits...

Reply:
Thanks for noticing that, will do it now!